remote_test.go

```diff
@@ -3,12 +3,13 @@ package blockstore
 import (
 	"context"
 	"fmt"
+	"math"
 	"math/rand"
-	"os"
 	"reflect"
 	"testing"
 	"time"
 
+	xtestutils "github.com/RTradeLtd/TxPB/v3/go/testutils"
```

Comment:
We should use the same package name as the directory name if possible. I suggest renaming the path to xtestutils

Reply:
good idea, let me do that